src/main/java/uk/gov/hmcts/darts/arm/mapper/impl/MediaArchiveRecordMapperImpl.java

```diff
@@ -230,20 +233,23 @@ private String getHearingDate(MediaEntity media) {
 
     private String getCaseNumbers(MediaEntity media) {
         String cases = null;
-        if (nonNull(media.getHearingList())) {
-            List<HearingEntity> hearings = media.getHearingList();
-            List<String> caseNumbers = hearings
-                    .stream()
-                    .map(HearingEntity::getCourtCase)
-                    .map(CourtCaseEntity::getCaseNumber)
-                    .toList();
+        if (CollectionUtils.isNotEmpty(media.getHearingList())) {
+            List<String> caseNumbers = media.getHearingList()
+                .stream()
+                .map(HearingEntity::getCourtCase)
+                .map(CourtCaseEntity::getCaseNumber)
+                .toList();
             cases = caseListToString(caseNumbers);
         }
         return cases;
     }
 
     private static String getCourthouse(MediaEntity media) {
-        return media.getCourtroom().getCourthouse().getCourthouseName();
+        String courthouse = null;
+        if (nonNull(media.getCourtroom().getCourthouse())) {
```

Comment:
do we need to check if media.getCourtroom() is null as well?

Reply:
Courtroon is not nullable, courthouse is nullable

Reply:
Not sure I follow your comment here @karen-hedges - courhouse link is also not null

```
                         Table "darts.courtroom"
     Column     |           Type           | Collation | Nullable | Default 
----------------+--------------------------+-----------+----------+---------
 cth_id         | integer                  |           | not null | 
 ```